### PR TITLE
Update to jakarta xml everywhere and update PDFbox

### PR DESCRIPTION
**Pull request by Siedlerchr**
I finally managed to upgrade to jakarta.xml stuff :)
Man is that complicated with all the jaxb stuff 

@koppor  I also refactored the XMP tests

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
TODO: Needs testing of the binaries. If they still work with import/export in jpackage

File: src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
 import org.apache.xmpbox.DateConverter;
 import org.apache.xmpbox.schema.DublinCoreSchema;
+import org.apache.xmpbox.type.BadFieldValueException;
+import org.tinylog.Logger;
```
Comment (calixtus) [inline src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java:30]: Why not slf4j?

**Review by calixtus — APPROVED** [review]
Codewise looks good to me!

**Comment by Siedlerchr** [comment]
tested locally using jpackage, seems to work fine.

**Pull request merged**